examples/frontend/index.js

```diff
@@ -1,8 +1,19 @@
-import  {Biscuit, PrivateKey, KeyPair, Fact, Rule, biscuit, authorizer, block} from '@biscuit-auth/biscuit-wasm';
+import {
+  Biscuit,
+  PrivateKey,
+  KeyPair,
+  Fact,
+  Rule,
+  biscuit,
+  authorizer,
+  block,
+} from "@biscuit-auth/biscuit-wasm";
 
 let keypair = new KeyPair();
 
-let pk = PrivateKey.fromString("473b5189232f3f597b5c2f3f9b0d5e28b1ee4e7cce67ec6b7fbf5984157a6b97");
+let pk = PrivateKey.fromString(
+  "473b5189232f3f597b5c2f3f9b0d5e28b1ee4e7cce67ec6b7fbf5984157a6b97"
```

Comment:
shouldn’t it take an algorithm prefix?

Reply:
yes, they should, I'm fixing that. As an aside, I think we should have different prefixes for private keys and public keys

Reply:
Yes, I was thinking about that the other day.

I'll open a PR on biscuit-rust

Reply:
https://github.com/eclipse-biscuit/biscuit-rust/pull/276 :)